demo.py

```diff
@@ -0,0 +1,146 @@
+class Resource(models.Model):
+    client = ForeignKey(Client)
+    owned_by = ForgeignKey(Resource, null=True)
+    name = CharField(blank=False, max_length=128)
+    configuration = JSONField(null=False)
```

Comment:
@antoviaque This `configuration` is meant to be used by subclasses to store arbitrary data: see earlier comment at https://github.com/open-craft/opencraft/pull/28/files#r44956742 . Admittedly that's not clear here :)